Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /celestia-app
 RUN make build
 
 # stage 2
-FROM alpine:3.17.1
+FROM alpine:3.17.2
```

Comment:
looks like the diff between 3.17.1 and 3.17.2 contains mostly openssl security fixes https://www.alpinelinux.org/posts/Alpine-3.17.2-released.html